OracleLinux/7/.env

```diff
@@ -0,0 +1,16 @@
+# Oracle Linux 7 configuration file
+#
+# Requires vagrant-env plugin
+#
+# This file will be overwritten on updates, it is recommended to make changes
+# in .env.local
+
+# Extensions
+# To extend the box, uncomment one of the following:
```

Comment:
I think this might be a bit confusing as users would probably uncomment the "`LAMP server`" heading/break.